tests/proxy_models/tests.py

```diff
@@ -105,6 +105,17 @@ def test_proxy_included_in_ancestors(self):
         with self.assertRaises(Person.MultipleObjectsReturned):
             StatusPerson.objects.get(id__lt=max_id + 1)
 
+    def test_multiple_abc(self):
```

Comment:
Can it be removed from the existing tests `test_no_base_classes` and `test_no_cbc` in this file (https://github.com/django/django/pull/6360)?